inst/extdata/install-packages.R

```diff
@@ -113,45 +92,134 @@ installEsqLabsR <- function() {
+    # excel file
+    # scenarioNames <- NULL
+
+    # Create `ScenarioConfiguration` objects from excel files
+    scenarioConfigurations <- readScenarioConfigurationFromExcel(
+      scenarioNames = scenarioNames,
+      projectConfiguration = projectConfiguration
+    )
+
+    scenarios <- createScenarios(scenarioConfigurations)
+
+    simulatedScenarios <- runScenarios(scenarios = scenarios)
+
+    cli_progress_done(result = "done")
+    cli_alert_success("Installation successful.")
+
+  },
+  error = function(error){
     cli_progress_done(result = "failed")
+    cli_alert_danger(text = paste("Simulation test failed: ", error))
+  })
+}
+
+initialize_project <- function(){
+
+  initialize_option <- utils::menu(c("Yes", "No"),
+                                   title = "Do you want to initialize project folder structure ?"
+  )
+
+  if (initialize_option == 1) {
+    cli_progress_step("Initialize project structure")
+    init_project(destination = "../")
+    update_project_conf_file()
+    cli_progress_done(result = "done")
   }
 
-  cli_alert_success("Installation successful.")
+}
+
+update_project_conf_file <- function(){
+  proj_name <- get_project_name()
+
+  timevalues_file <-
+    list.files("../Data",
+               pattern = "*TimeValuesData.xlsx")[1]
 
+  compoundpropertiesinternal_file <-
+    list.files("../Data",
+               pattern = "*_Compound Properties \\(Internal\\).xlsx")[1]
+
+  project_configuration <-
+    createDefaultProjectConfiguration(path = "../ProjectConfiguration.xlsx")
+
+  project_configuration$dataFile <- timevalues_file
+
+  project_configuration$compoundPropertiesFile <- compoundpropertiesinternal_file
+
+  project_configuration$save()
+}
+
+get_project_name <- function(){
+  rproj_name <- list.files(pattern = ".Rproj")
+  proj_name <- sub(pattern = "_V.*?.Rproj",
+                   replacement = "",
+                   x = rproj_name)
+
+  return(proj_name)
+}
+
+restart_rstudio <- function() {
+  rm(list = ls(envir = .GlobalEnv), envir = .GlobalEnv)
   # if running in RStudio, restart R session
   if (isAvailable()) {
     cli_alert_info("Restarting R session.")
-    rm(list = ls(envir = .GlobalEnv))
-    rstudioapi::restartSession(command = "library(esqlabsR)")
+    restartSession(command = "library(esqlabsR)")
   } else {
     cli_alert_info("Please restart R to complete the installation.")
   }
 }
 
-# run installation function
-installEsqLabsR()
+setup_esqlabsR <- function() {
+  # Display a menu asking if user wants to install packages in local env or  globally
+  install_option <- utils::menu(c("In local environment (available for one project)", "In Glocal environment (available for all projects)"),
+                                title = "Where do you want to install {esqlabsR} and other packages?"
+  )
+
+  install_script_deps(install_option)
+
+  check_RTools()
+
+
+  get_esqlabsR()
+
+  get_PKSim_Minimal(install_option)
+
+  load_esqlabsR()
```

Comment:
With local installation, I am always getting the error:

```
✔ Verify PKSim installation [5.2s]
Loading required package: ospsuite
Loading required package: rClr
Loading the dynamic library for Microsoft .NET runtime...
Loaded Common Language Runtime version 4.0.30319.42000

Error in value[[3L]](cond) : 
  Package ‘ospsuite’ version 11.2.260 cannot be unloaded:
 Error in unloadNamespace(package) : namespace ‘ospsuite’ is imported by ‘esqlabsR’, ‘ospsuite.parameteridentification’ so cannot be unloaded
In addition: Warning messages:
1: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
2: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
3: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
4: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
5: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
6: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
7: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
8: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
9: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
10: In untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
✖ Load esqlabsR [64ms]
```

After installation from GitHub, the package is already loaded and "locked", and an attempt to load it again fails.

Reply:
1. Did you try to run the script in a new Project (not in package directory)
2. Did you remove esqlabsR prior installation ?
3. Did you restart R (Ctrl + shit + F10) completely between two attempts ?

The ` initialize_project()` part cannot work since the github version do not have the "destination" argument for init_project().

However, you can test this function by loading the branch version of the package.